frontend/src/metabase/palette/components/Palette.module.css

```diff
@@ -0,0 +1,17 @@
+.input {
+  padding: var(--mantine-spacing-lg);
+  padding-left: 60px;
+  font-size: 17px;
```

Comment:
such a weird number to have. maybe we should just use `1rem` here? otherwise, feel free to resolve this comment.